### PR TITLE
Santa's Helpers' Helpers

### PR DESCRIPTION
**Pull request by GeekRuthie**
Here's the draft of my article!

### CODE REVIEW

**Comment by oalders** [comment]
Closes #289

File: 2023/incoming/santa-s-helpers-helpers.pod [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+=item L<Test::DBIx::Class>
+
+The grumpy testing elves found this module very useful; it let them set up and
+run unit and integration tests in a known-clean enviroment. They designed the
+test fixtures and wrote tests to make sure every function behaved as it was
+supposed to.
+
+=item L<Dancer2::Plugin::DBIx::Class>
+=item L<DBIx::Class::ResultSetNames>
```
Comment (openstrike) [inline 2023/incoming/santa-s-helpers-helpers.pod:83]: These 2 =item lines need a blank line between them. Alternatively just make the two modules into one =item:

`=item L<Dancer2::Plugin::DBIx::Class> and L<DBIx::Class::ResultSetNames>`

File: 2023/incoming/santa-s-helpers-helpers.pod [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+their naughty/nice ratio.  The heart of it lay in a single line of code:
+
+=begin perl
+
+   db_person->born_before($dt_1_year_ago)->update({ auto_nice => 0 });
+
+=end perl
+
+=item L<DBIx::Class::Helpers>
+=item L<DBIx::Class::MoreHelpers>
```
Comment (openstrike) [inline 2023/incoming/santa-s-helpers-helpers.pod:110]: Same here: separate with a blank line or else combine into one =item

**Review by openstrike — CHANGES_REQUESTED** [review]
Very nicely written. Just an issue with the POD as detailed here.

**Review by openstrike — APPROVED** [review]
Perfect.

**Pull request merged**